commit 74184586228af6c362f970c84fce58da3fcbdec8
Author: Matt Oliver <protogonoi@gmail.com>
Date:   Tue Apr 4 00:34:47 2017 +1000

    fix memory errors when using gas processor with coff object output.

diff --git a/modules/objfmts/coff/coff-objfmt.c b/modules/objfmts/coff/coff-objfmt.c
index 6fcb0e3c..8f753bfe 100644
--- a/modules/objfmts/coff/coff-objfmt.c
+++ b/modules/objfmts/coff/coff-objfmt.c
@@ -290,6 +290,7 @@ coff_common_create(yasm_object *object)
     objfmt_coff->done_prolog = 0;
     objfmt_coff->unwind = NULL;
     objfmt_coff->ssym_imagebase = NULL;
+    objfmt_coff->def_sym = NULL;
 
     return objfmt_coff;
 }
